### ISSUE TITLE
NTR aerophagia, air-swallowing

### ISSUE THREAD

**Issue by Daniel-Olson** [obophenotype/human-phenotype-ontology#10005]
**Preferred term label:**
aerophagia

**Synonyms**
aerophagy, air-swallowing

**Definition (free text, please give PubMed ID)**
excessive swallowing of air or gas.

**Parent term (use hpo.jax.org/app)**
(HP:0025032) Abnormality of digestive system physiology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-8134-1207

**Comment by pnrobinson** [comment]
Thanks!
@Daniel-Olson 
Adding new term: 
- ID: HP:6000060
- label: Aerophagia
- parents: [Abdominal symptom (HP:0011458)](https://hpo.jax.org/app/browse/term/HP:0011458)
- pmid: PMID:12095473
- def: Repetitive air swallowing and belching and that may result in abdominal distention.
- comment: 
- orcid: orcid:0000-0002-8134-1207
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10005
- synonyms:
